### ISSUE TITLE
[BE] Profile: Are we sending deprecated IDs to VA Profile?

### ISSUE THREAD

**Issue by Samara-Strauss** [department-of-veterans-affairs/va.gov-team#17739]
## Background

[Erik made the following comment in Slack](https://dsva.slack.com/archives/C909ZG2BB/p1608220731228000?thread_ts=1608216577.222500&cid=C909ZG2BB):

> On the identifier front, the "deprecated" errors we are seeing have to do with us (I think our vets-api layer) sending a deprecated VA Profile ID when making an update. So a user can sometimes have more than one VA Profile ID associated with them and for some reason we are using a deprecated one. I think that explains why users can sign in fine. I'm sure we don't care about their VA Profile IDs when people sign in.

This came out of some investigation we were doing into why error counts were so high. As a result, we have a few questions:

- Why we are sometimes sending deprecated IDs to VA Profile? 
- Is this a larger identity issue? And, if so, do we need to bring in someone from the identity team?

## Tasks

- [ ] Look into why we are sending deprecated IDs to VA Profile sometimes.
- [ ] Figure out if this is a larger identity issue, and if we need to bring this to the attention of someone on the identity team.

**Comment by lihanli** [comment]
@erikphansen is this the error you were seeing? 
```
code: PERS203, 
key: vaprofileId.Deprecated, 
potentially_self_correcting_on_retry: False, 
severity: ERROR, 
text: The given id for contact information is deprecated 3,540,179
```
http://sentry.vfs.va.gov/organizations/vsp/issues/15749/events/0899e03404464fe59baa42c1011901f3/?environment=production&project=3&query=is%3Aunresolved+level%3Aerror+controller_name%3Aaddresses+vet360%3Ageneral_client_error&statsPeriod=14d

**Comment by erikphansen** [comment]
@lihanli I don't think so. We were just looking at [the top errors in GA](https://analytics.google.com/analytics/web/#/dashboard/pq_-PrkvQleUdCBbV7eq7Q/a50123418w177519031p176188361/) and the deprecated errors we were getting from `vets-api` contained `MVI207` and `MVIIdentifierDeprecated `.

**Comment by mshea0606** [comment]
@mshea0606 to ask about MVI207 errors in VA Profile channel

**Comment by mshea0606** [comment]
ian mcewen said they have everything they need and they will be working on this in sprint 42 [slack](https://dsva.slack.com/archives/C7TE0PFTL/p1613676887099500?thread_ts=1611332694.003100&cid=C7TE0PFTL)

**Comment by mshea0606** [comment]
@mshea0606 following up on this [Slack](https://dsva.slack.com/archives/C7TE0PFTL/p1608215573214600?thread_ts=1608135206.197100&cid=C7TE0PFTL)

**Comment by Samara-Strauss** [comment]
Asked in Slack, but it is unclear what work still needs to happen on our end and what next steps are for both our team and VA Profile. This ticket has been open for a while, so please do what you need to do to move it forward.

**Comment by mshea0606** [comment]
We are currently waiting on a response from VA Profile

**Comment by Samara-Strauss** [comment]
@mshea0606 can you confirm with @erikphansen and @lihanli whether there is any more work for them to do on this issue? I don't think there is, and if not, we can close this ticket.

If there is no more work for Erik or Lihan, please create a new ticket to track VA Profile's work, and please include the relevant parts of [this thread](https://dsva.slack.com/archives/C7TE0PFTL/p1608138662207400) so we have context for what's going on and what we can expect next from VA Profile.

**Comment by erikphansen** [comment]
I'm 99% sure that there's nothing the FE can do about this issue, period.

**Comment by lihanli** [comment]
nothing else for the backend to do right now

**Comment by Samara-Strauss** [comment]
Great. I'll close this and @mshea0606 can create the necessary follow-up ticket to track VA Profile work and correspondence.
